story-generator/src/app/app.css

```
@@ -1,759 +1,593 @@
+  max-height: 420px;
+  overflow-y: auto;
 }
 
-/* Progress and Messages */
-.audio-progress {
-  background: white;
-  padding: 1.5rem;
-  border-radius: 10px;
-  margin-top: 1rem;
-  text-align: center;
+.chapter-content ::ng-deep p {
```

Stylelint (17.12.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)